inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/codec/ProtocolEncoder.java

```diff
@@ -287,8 +288,8 @@ private ByteBuf writeToBuf5(EncodeObject object) {
                         EncryptInfo encryptInfo = encryptEntry.getRsaEncryptInfo();
                         msgAttrs = msgAttrs + "_userName=" + object.getUserName()
                                 + "&_encyVersion=" + encryptInfo.getVersion()
-                                + "&_encyDesKey=" + encryptInfo.getRsaEncryptedKey();
-                        body = EncryptUtil.desEncrypt(body, encryptInfo.getDesKey());
+                                + "&_encyAesKey=" + encryptInfo.getRsaEncryptedKey();
+                        body = EncryptUtil.aesEncrypt(body, encryptInfo.getDesKey());
```

Comment:
It is not fit that getDesKey is a AES Key.
body = EncryptUtil.aesEncrypt(body, encryptInfo.getDesKey());